CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# 0.23.0
+
+- The `propose-update` has been deprecated in favour of `propose-downstream`
```

Comment:
```suggestion
- The `propose-update` has been renamed to `propose-downstream`; `propose-update` is now deprecated
```

Reply:
oh, it's 2 lines actually, so my suggestion doesn't make sense; my point here is that we renamed the command; not sure if my suggestion will be more clear to users

Reply:
Yes, I think Tomas's version is a bit more clearer